src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpInteractiveCommands.cs

```diff
@@ -90,32 +91,32 @@ public void VerifyUndoAndRedo()
             SendKeys(Ctrl(VirtualKey.Y));
             VerifyLastReplInput(" 2 + 4 ");
             SendKeys(VirtualKey.Enter);
-            WaitForReplOutput("6");
+            this.WaitForLastReplOutput("6");
         }
 
         [Fact]
         public void CutDeletePasteSelectAll()
         {
             SendKeys("Text");
-            this.ExecuteCommand("Edit.LineStart");
-            this.ExecuteCommand("Edit.LineEnd");
-            this.ExecuteCommand("Edit.LineStartExtend");
-            this.ExecuteCommand("Edit.SelectionCancel");
-            this.ExecuteCommand("Edit.LineEndExtend");
-            this.ExecuteCommand("Edit.SelectAll");
-            this.ExecuteCommand("Edit.SelectAll");
-            this.ExecuteCommand("Edit.Copy");
-            this.ExecuteCommand("Edit.Cut");
-            this.ExecuteCommand("Edit.Paste");
-            this.ExecuteCommand("Edit.Delete");
-            this.ExecuteCommand("Edit.LineUp");
-            this.ExecuteCommand("Edit.LineDown");
-            this.ExecuteCommand("Edit.Paste");
-            this.ExecuteCommand("Edit.Paste");
+            ExecuteCommand(WellKnownCommandNames.Edit_LineStart);
```

Comment:
This series of command executions seems really fishy. What is this test actually asserting?